src/activeobject.h

```diff
@@ -72,6 +73,33 @@ enum ActiveObjectCommand {
 	AO_CMD_SET_ANIMATION_SPEED
 };
 
+struct BoneOverride
+{
+	struct PositionProperty
+	{
+		v3f previous{};
```

Comment:
Initializer list is not necessary for classes with default constructors, i.e. `v3*` and `core::quaternion`.